biojava-structure/src/main/java/org/biojava/nbio/structure/symmetry/core/RotationSolver.java

```diff
@@ -21,12 +21,7 @@
 
 package org.biojava.nbio.structure.symmetry.core;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
This is dangerous. Unless the rule has changed, the norm is to import the exact files/classes you need only.